### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by indirap**

### CODE REVIEW

**Comment by rnowosielski** [comment]
Any clue why the build are failing here ?

**Comment by jboerjes** [comment]
The issue is that in the package.json, the version is 0.1 and not 0.0.1 or 0.0.2 (clearly an update is needed) ... and apparently npm doesn't like that version.

Everything builds locally if I set the version to something with 3 parts.

**Comment by ro-tex** [comment]
NPM requires semver: https://docs.npmjs.com/misc/semver

**Comment by indirap** [comment]
Sorry for the late response; will fix today.

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ script:
 - npm run code-check
 - npm run cover
 - grunt build
+before_deploy:
+- cd dist
```
Comment (indirap) [inline .travis.yml:11]: This should fix #8 where @jboerjes found that the deployed package hasn't been transpiled to ES5

File: package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
-  "name": "fulfillment-location-client",
+  "name": "cimpress-fulfillment-location",
```
Comment (indirap) [inline package.json:2]: Fixes #7

**Review by jboerjes — APPROVED** [review]
looks good.

**Comment by indirap** [comment]
This repo has been re-published on NPM under [`cimpress-fulfillment-location`](https://www.npmjs.com/package/cimpress-fulfillment-location) and I've added the trdlnik team members to that package.

I've also marked the old package [`fulfillment-location-client`](https://www.npmjs.com/package/cimpress-fulfillment-location) as deprecated.

**Pull request merged**